src/index.js

```diff
@@ -6,6 +6,8 @@ import ext from './ext';
 import muiSetup from './mui-setup';
 import { render, teardown } from './table/root';
 import manageData from './table/handle-data';
+// This line is replaced by rollback with an import for internal builds
+const __OPIONAL_THEME_DEPS__ = {}; // eslint-disable-line no-underscore-dangle
```

Comment:
I don't know why you use __ before/after the variable name, does it mean it has to do with building or something?

Reply:
yea, more or less like a warning "DON*T TOUCH!".